assets/styles/css/main.css

```diff
@@ -211,6 +216,62 @@ body {
   height: 200px;
 }
 
+.menu-container {
+  width: 70%;
+  height: 200px;
+  z-index: 2;
+  position: absolute;
+  background-color: var(--color-white);
+  box-shadow: 3px 3px rgba(226, 226, 226, 0.603);
+}
+
+.menu-items {
+  height: 100%;
+  margin-block-start: unset;
+  margin-block-end: unset;
+  padding-inline-start: unset;
+  padding: 1.2rem 0 1.2rem 1rem;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.links-menu {
+  text-decoration: none;
+  font-size: 1rem;
+  color: var(--color-text);
+  font-weight: 700;
+  margin-top: 2rem;
+}
+
+.links-menu:hover {
+  color: var(--color-main);
+}
+
+.links-active-menu {
+  color: var(--color-main);
+}
+
+.btn-ticket-menu {
+  border: 3px solid var(--color-main);
+  border-radius: 5px;
+  padding: 0.3rem 1.3rem;
+  text-decoration: none;
+  font-size: 1rem;
+  color: var(--color-main);
+  font-weight: 700;
+}
+
+.btn-ticket-menu:hover {
+  color: var(--color-white);
+  background: var(--color-main);
+}
+
+.show-or-hide {
+  visibility: hidden;
+}
+
 @media only screen and (max-width: 768px) {
   *,
   ::after,
```

Comment:
- [ ]  The CSS styling should be built as **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around. Here and in other parts of your css code, you have not developed with "mobile first" approach. Please, let's correct it.

Reply:
Hello @george-shammar , Thanks for giving me the feedback. I am going to work on it.